File: examples/swagger-2/src/test/resources/asserts/typescript/typescript_feign_func/src/model/paging/Page.ts
/* tslint:disable */
/* eslint-disable */

        import {Pageable} from "./Pageable";
        import {Sort} from "./Sort";



export interface  Page<T> {

        content: Array<T>;
        empty: boolean;
        first: boolean;
        last: boolean;
        number: number;
        numberOfElements: number;
        size: number;
        totalElements: number;
        totalPages: number;
        pageable: Pageable;
        sort: Sort;
}

File: examples/swagger-2/src/test/resources/asserts/java/retrofit/src/com/wuxp/codegen/swagger2/model/paging/Page.java
package com.wuxp.codegen.swagger2.model.paging;

import lombok.Data;
import lombok.experimental.Accessors;
import jakarta.validation.constraints.*;
import com.fasterxml.jackson.annotation.JsonFormat;
        import com.wuxp.codegen.swagger2.model.paging.Pageable;
        import com.wuxp.codegen.swagger2.model.paging.Sort;

@Data
@Accessors(chain = true)
public class  Page<T> {

        private List<T> content;

        private String empty;

        private String first;

        private String last;

        private Integer number;

        private Integer number_of_elements;

        private Integer size;

        private Integer total_elements;

        private Integer total_pages;

        private Pageable pageable;

        private Sort sort;

}
